### DISCUSSION TITLE
FWHM of ConvGaussianDoniachSinglett

### DISCUSSION THREAD

**Discussion by Hexanders** [Julian-Hochhaus/lmfitxps#10] [Q&A] [answered]
I have a question about the FWHM of Lorentzian. 
In the model  ConvGaussianDoniach it is given as lorentzian_fwhm: = 'sigma*(2+gamma*2.5135+ (gamma*3.6398)**4)'

I am a bit confused. Where did this expression come from? It is probably trivial and I know if you let gamma go to 0, Doniach 
became Lorentzian and FWHM is 2*sigma. 
But how do you get this expression from above? 
And also expression for FWHM of Gaussian '2*gaussian_sigma*1.1774'.  

It would be nice if you could give me a hint.

**Comment by Julian-Hochhaus** [comment, marked as answer]
> I have a question about the FWHM of Lorentzian. In the model ConvGaussianDoniach it is given as lorentzian_fwhm: = 'sigma*(2+gamma_2.5135+ (gamma_3.6398)**4)'
> 
> I am a bit confused. Where did this expression come from? It is probably trivial and I know if you let gamma go to 0, Doniach became Lorentzian and FWHM is 2_sigma. But how do you get this expression from above? And also expression for FWHM of Gaussian '2_gaussian_sigma*1.1774'.

The second one is easy: 
![image](https://wikimedia.org/api/rest_v1/media/math/render/svg/09da3bf013e549b039044a2287c251512706e402)
For details, see [Wikipedia](https://en.wikipedia.org/wiki/Voigt_profile). 
1.1774~sqrt(2*ln(2)).

The first one should be taken similar to the FWHM of the Voigt profile. Similar to the FWHM of the Voigt profile, there exists no closed equation for the FWHM of Doniach. I therefore did a fit on how I needed to modify the equation of the FWHM of the Lorentzian depending on the asymmetry of the Doniach profile (gamma) to get a good approximation of the FWHM of the Doniach profile.
The given equation works with an error <1% in the interesting range (0<=gamma<=0.4) and especially in the most common range (0<=gamma<=0.2) the error is smaller than 0.2%.

However, comparable to the FWHM of the Voigt, no closed equation exists. I therefore followed a similar approach to multiple papers which did fits for the FWHM of the Voigt, but in my case for the Doniach profile.
Compare e.g. [https://www.sciencedirect.com/science/article/pii/0022407377901613?via%3Dihub](https://www.sciencedirect.com/science/article/pii/0022407377901613?via%3Dihub)

**Reply by Hexanders** [reply]
Thank you very much. 
Did you have plans to publish your results recurring the FWHM of Doniach?
I think it would help some people out there!

**Reply by Julian-Hochhaus** [reply]
at least I plan to write a section in the manual as soon as possible :)
